src/framework/ui/dropdown/dropdownMenu.component.tsx

```diff
@@ -68,6 +68,7 @@ export class DropdownMenu extends React.Component<DropdownMenuProps> {
     ) : (
       <DropdownItem
         {...info}
+        {...info.item}
```

Comment:
Is it expected that a final object of properties will look like the one below?
```js
{
  item: {
    ...itemProps, // <-- props of the `item` 
  },
  ...itemProps, // <-- same props without nesting in `item`
  index,
  // ...
}
```

Reply:
Refactored